webapp/src/components/testing/Testing.vue

```diff
@@ -232,17 +245,32 @@ onActivated(async () => {
 })
 
 const downloadModel = async (task: Task): Promise<void> => {
-  const client = new clients.Local(CONFIG.serverUrl, task, aggregator.getAggregator(task))
-  const model = await client.getLatestModel()
-  const source = {
-    type: 'saved' as const,
-    taskID: task.id,
-    name: task.trainingInformation.modelID,
-    tensorBackend: task.trainingInformation.tensorBackend,
-  }
-  await memory.value.saveModel(source, model)
-  await memoryStore.initModels()
+  try {
+    const client = new clients.Local(CONFIG.serverUrl, task, aggregator.getAggregator(task))
+    const model = await client.getLatestModel()
+    const source = {
+      type: 'saved' as const,
+      taskID: task.id,
+      name: task.trainingInformation.modelID,
+      tensorBackend: task.trainingInformation.tensorBackend,
+    }
+    await memory.value.saveModel(source, model)
+    await memoryStore.initModels()
+    toaster.success("Model successfully downloaded!")
+    const scrollableDiv = document.getElementById('scrollable-div')
+    if (scrollableDiv !== null) {
+      scrollableDiv.scrollTo({
+        top: 0,
+        left: 0,
+        behavior: "smooth",
+      });
+    }
+  } catch (e) {
+    toaster.error("Something went wrong, please try again later.")
```

Comment:
I think it's bad practice to dump error messages in a user feedback, in particular it can comprise security and privacy. If the error can be solved user-side we should handle the error and state potential solutions, if it can't be then I don't think we should display the error (like a TypeError is not useful to show if we already tell them it's on our side)

Reply:
make sense, thanks for the input!